src/app/cli/src/explore/api_server.rs

```diff
@@ -175,6 +173,10 @@ impl APIServer {
             None
         };
 
+        router = router
+            .layer(Extension(gql_schema))
+            .layer(Extension(api_server_catalog));
+
```

Comment:
Can you explain the change in layering order here?

Reply:
I needed to move the extensions' declaration below because I started using `catalog` in the freshly-added `set_system_time_handler()`.

Otherwise, `catalog` could not be found.

---

A fair question arises, why was `gql_schema` also moved? I thought it was better to have both extensions side by side.